base_layer/core/src/mempool/mempool_storage.rs

```diff
@@ -88,6 +88,10 @@ impl MempoolStorage {
                 );
                 let timer = Instant::now();
                 let weight = self.get_transaction_weighting();
+                if tx.body.get_total_fee().as_u64() < self.unconfirmed_pool.config.min_fee {
+                    warn!(target: LOG_TARGET, "Tx: ({}) fee too low, rejecting",tx_id);
```

Comment:
Warn is too high. It's not a threat if the fees are too low, and you don't want the warning channel to be spammed if someone is flooding with zero-fee txs.
Info, or even debug should be fine.